### PR TITLE
docs(grid):update autofit grid kb

### PR DESCRIPTION
**Pull request by NansiYancheva**
Related to: https://github.com/telerik/blazor/issues/9117

### CODE REVIEW

File: knowledge-base/grid-autofit-columns.md [hunk 1]
```diff
@@ -37,12 +37,15 @@ Autofitting columns on initial load is a current limitation of the built-in auto
...
+There are two possible implementations to work this around and autofit the Grid columns as the component initializes. They depend on whether the Grid is data-bound with [`Data` parameter](#data-parameter) or [`OnRead` event](#onread-event). In both cases the Grid [`Resizable` parameter]({%slug components/grid/columns/resize%}) has to be `true`.
 
 ### Data Parameter
 
-To AutoFit the Grid columns on initial load of the component you have to use a provision like the `MutationObserver`. This JavaScript tool notifies about DOM changes. The code snippet below uses the MutationObserver to trigger the `AutoFitAllColumns` method when the nodes in the content of Grid have mutated (rendered in this case). 
-
+To AutoFit the Grid columns on initial load of the component you have to:
+* Use the Grid [`Class` parameter]({%slug grid-overview%}#grid-parameters) to identify the Grid that you want to autofit.
+* When the Grid is rendered for the first time, call a JS function in the `OnAfterRenderAsync` method.
+* The JS function `observeTarget` in the second tab in the example below sets up the `MutationObserver`. This is a JavaScript tool, which listens for DOM changes.
```
Comment (ikoevska) [inline knowledge-base/grid-autofit-columns.md:47]: This bullet item doesn't relate to the others - it's not something that the developer needs to do. It provides additional information for the last bullet point on the list. 

Consider reworking the last two bullet points into something like this:

```
* Listen for DOM changes and trigger the `AutoFitAllColumns` method when the nodes in the content of the Grid have mutated.

The **JavaScript** tab in the following example shows how to use an `observeTarget` JS function to set up the `MutationObserver` tool to listen for DOM changes.
```

**Review by ikoevska — APPROVED** [review]

File: knowledge-base/grid-autofit-columns.md [hunk 2]
```diff
@@ -37,12 +37,15 @@ Autofitting columns on initial load is a current limitation of the built-in auto
...
 
 ### Data Parameter
 
-To AutoFit the Grid columns on initial load of the component you have to use a provision like the `MutationObserver`. This JavaScript tool notifies about DOM changes. The code snippet below uses the MutationObserver to trigger the `AutoFitAllColumns` method when the nodes in the content of Grid have mutated (rendered in this case). 
-
+To autofit the Grid columns on the initial load of the component:
+* Use the Grid [`Class` parameter]({%slug grid-overview%}#grid-parameters) to identify the Grid that you want to autofit.
+* When the Grid is rendered for the first time, call a JS function in the `OnAfterRenderAsync` method.
+* The JS function `observeTarget` in the second tab in the example below sets up the `MutationObserver`. This is a JavaScript tool, which listens for DOM changes.
+* Use the `MutationObserver` to trigger the `AutoFitAllColumns` method when the nodes in the content of Grid have mutated (rendered in this case).
```
Comment (dimodi) [inline knowledge-base/grid-autofit-columns.md:48]:
```suggestion
* Use the `MutationObserver` to trigger the `AutoFitAllColumns` method when the table rows in the Grid have rendered.
```

File: knowledge-base/grid-autofit-columns.md [hunk 3]
```diff
@@ -37,12 +37,15 @@ Autofitting columns on initial load is a current limitation of the built-in auto
 
 While the autofitting feature includes fitting the headers, it also targets the data cells and the footers, so it needs all the rows in the process.
 
-There are two possible implementations to work this around and autofit the Grid columns as the component initializes. They depend on whether the Grid is data-bound with [`Data` parameter](#data-parameter) or [`OnRead` event](#onread-event).
+There are two possible implementations to work this around and autofit the Grid columns as the component initializes. They depend on whether the Grid is data-bound with the [`Data` parameter](#data-parameter) or the [`OnRead` event](#onread-event). In both cases, the Grid [`Resizable` parameter]({%slug components/grid/columns/resize%}) has to be `true`.
```
Comment (dimodi) [inline knowledge-base/grid-autofit-columns.md:40]:
```suggestion
There are two possible implementations to work this around and autofit the Grid columns as the component initializes. They depend on whether the Grid is data-bound with the [`Data` parameter](#data-parameter) or the [`OnRead` event](#onread-event). In both cases, the Grid [`Resizable` parameter]({%slug components/grid/columns/resize%}) must be set to `true`.
```

File: knowledge-base/grid-autofit-columns.md [hunk 4]
```diff
@@ -37,12 +37,16 @@ Autofitting columns on initial load is a current limitation of the built-in auto
...
+There are two possible implementations to work this around and autofit the Grid columns as the component initializes. They depend on whether the Grid is data-bound with the [`Data` parameter](#data-parameter) or the [`OnRead` event](#onread-event). In both cases, the Grid [`Resizable` parameter]({%slug components/grid/columns/resize%}) must be set to `true`.
 
 ### Data Parameter
 
-To AutoFit the Grid columns on initial load of the component you have to use a provision like the `MutationObserver`. This JavaScript tool notifies about DOM changes. The code snippet below uses the MutationObserver to trigger the `AutoFitAllColumns` method when the nodes in the content of Grid have mutated (rendered in this case). 
+To autofit the Grid columns on the initial load of the component:
+* Use the Grid [`Class` parameter]({%slug grid-overview%}#grid-parameters) to identify the Grid that you want to autofit.
+* When the Grid is rendered for the first time, call a JS function in the `OnAfterRenderAsync` to observe the Grid.
+* Listen for DOM changes using JS and invoke the `AutoFitAllColumns` method when the table rows in the Grid have rendered.
```
Comment (dimodi) [inline knowledge-base/grid-autofit-columns.md:47]:
```suggestion
To autofit the Grid columns on the initial load of the component:

* Use the Grid [`Class` parameter]({%slug grid-overview%}#grid-parameters) to identify the Grid that you want to autofit.
* When the Grid is rendered for the first time, call a JavaScript function in `OnAfterRenderAsync` to observe the Grid.
* Listen for DOM changes using JavaScript and invoke the `AutoFitAllColumns` method after the table rows in the Grid have rendered.
```

**Pull request merged**